travis.sh

```diff
@@ -180,7 +183,7 @@ function prepare_ros_workspace() {
          http://* | https://* | file://*) ;; # use url as is
          *) item="file://$CI_SOURCE_PATH/$item" ;; # turn into proper url
       esac
-      travis_run_true wstool merge -k $item
+      travis_run_true wstool merge -r -y $item
```

Comment:
This changes semantics from keeping existing packages to replacing them with new ones.
Can you better motivate this change? Did you observed issues with the previous approach?

Reply:
I am still working on getting moveit_ci to build off of the source image of moveit with a pre-built master branch. It seems to me that this is the intuitive behavior. 

If there is an existing rosinstall file from moveit, but you want to build against a different version of one of the dependencies or even a different version of moveit, why would we keep the pre-built repo? 

I don't know why we would want to overwrite a user's `UPSTREAM_REPOSITORIES` with the moveit.rosinstall ones but maybe I am overlooking something